modules/minio/src/main/java/org/testcontainers/containers/MinIOContainer.java

```diff
@@ -0,0 +1,127 @@
+package org.testcontainers.containers;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+/**
+ * Testcontainers implementation for MinIO.
+ * <p>
+ * Supported image: {@code MinIO}
+ * <p>
+ * Exposed ports: 9000,9001
+ */
+@Slf4j
+public class MinIOContainer extends GenericContainer<MinIOContainer> {
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("minio/minio");
+
+    private static final String DEFAULT_TAG = "latest";
+
+    private static final int MINIO_S3_PORT = 9000;
+
+    private static final int MINIO_UI_PORT = 9001;
+
+    private static final String DEFAULT_USER = "miniouser";
+
+    private static final String DEFAULT_PASSWORD = "miniopassword";
+
+    private String userName = DEFAULT_USER;
+
+    private String password = DEFAULT_PASSWORD;
+
+    /**
+     * Constructs a MinIO container with the latest tag
+     */
+    public MinIOContainer() {
+        this(DEFAULT_IMAGE_NAME.withTag(DEFAULT_TAG));
+    }
```

Comment:
default constructor should not be provided. image should be always provided by the user.